.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```

Comment:
dunno if this is needed?

Reply:
If, like me, your global npmrc points to a private registry, this makes sure that thie project uses the standard one, else the lock file will point to locations inaccessible to others.

Reply:
thought the point of `.npmrc` is to localise project-specific config not the other way around 😄 

not a problem either way though